pkg/util/output/progress_bar.go

```diff
@@ -0,0 +1,28 @@
+package output
+
+import (
+	"os"
+
+	"github.com/schollz/progressbar/v3"
+)
+
+func GetProgressBar(filesLength int) *progressbar.ProgressBar {
+	return progressbar.NewOptions(filesLength,
+		progressbar.OptionSetWriter(os.Stderr),
+		progressbar.OptionShowCount(),
+		progressbar.OptionEnableColorCodes(false),
+		progressbar.OptionShowElapsedTimeOnFinish(),
+		progressbar.OptionOnCompletion(func() {
+			PlainLogger(os.Stderr).Msgf("\n")
+		}),
+		progressbar.OptionShowIts(),
+		progressbar.OptionSetItsString("files"),
+		progressbar.OptionSetDescription("scanning repository..."),
```

Comment:
```suggestion
		progressbar.OptionSetDescription("Scanning repository %s...", path),
```

How about this?
Obviously we would need to know the path :)

Reply:
perhaps in a line above it?
 think it might be too long when using absolute paths.
```
scanning repository  /Users/vjeranfistric/go/src/github.com/bearer/broker...   7% [==>                                     ] (357/4492, 121 files/s) [2s:34s]

```

Reply:
also just realised we should probably turn off progress bar when debug is on? otherwise it will produce logs like this:
```
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"balancer spawning monitor....."}
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"balancer got task request"}
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"balancer directly offloading work to worker"}
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"balancer got 0 number of workers online"}
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"balancer spawning worker"}
{"level":"debug","time":"2022-10-21T14:58:01+02:00","message":"spawning on port :59296"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"started scan processing"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"running scan worker on port `:59296`"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"0d0dc5b3-9087-4ef7-be2b-4606058cf34c spawned after 0.72 seconds"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"worker uuid 6f657b65-9e3e-4b56-b771-3133138682e4 working on repo /Users/vjeranfistric/go/src/github.com/bearer/broker"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"skipping file due to file skip rules: /dist/worker"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"skipping file due to file skip rules: /worker"}
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"ignored due to file type /Users/vjeranfistric/go/src/github.com/bearer/broker/.DS_Store"}
scanning repository...   0% [                                        ] (1/4492, 97 files/s) [0s:46s]{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"processing file /Users/vjeranfistric/go/src/github.com/bearer/broker/.dockerignore"}
scanning repository...   0% [                                        ] (2/4492, 96 files/s) [0s:46s]{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"ignored due to file type /Users/vjeranfistric/go/src/github.com/bearer/broker/.envrc"}
scanning repository...   0% [                                        ] (3/4492, 100 files/s) [0s:44s]{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"processing file /Users/vjeranfistric/go/src/github.com/bearer/broker/.envrc.example"}
scanning repository...  48% [==================>                     ] (2162/4492, 115 files/s) [18s:20s]
{"level":"debug","time":"2022-10-21T14:58:02+02:00","message":"ignored due to file type /Users/vjeranfistric/go/src/github.com/bearer/broker/.DS_Store"}
scanning repository...  49% [==================>                     ] (2163/4492, 115 files/s) [18s:20s]
```

Reply:
@cfabianski I've pushed some updates which take care of both `--debug interaction` as well as showing the target path, it is in pr description